fdbserver/VersionedBTree.actor.cpp

```diff
@@ -754,28 +778,33 @@ class FIFOQueue {
 			pageID = newPageID;
 			offset = newOffset;
 
-			if (BUGGIFY) {
-				// Randomly change the byte limit for queue pages.  The min here must be large enough for at least one
-				// queue item of any type.  This change will suddenly make some pages being written to seem overfilled
-				// but this won't break anything, the next write will just be detected as not fitting and the page will
-				// end.
-				queue->dataBytesPerPage =
```

Comment:
Is this removal intentional?

Reply:
Yes, this works differently now.  Queue pages store explicitly how much space for items they have, called `itemSpace` and when the queue page is created the `itemSpace` is randomly reduced with a BUGGIFY which serves the same purpose as this.